bin/pip.py

```diff
@@ -684,6 +705,8 @@ def _run_setup_file(filename):
                 print("Warning: pkg_resources not available, skipping entry_points definitions.")
                 entry_points = {}
         for epn in entry_points:
+            if self.verbose:
+                print("Handling entrypoints for: " + epn)
             ep = entry_points[epn]
             if isinstance(ep, (str, unicode)):
```

Comment:
if isinstance(ep, basestring):

Reply:
`basestring` doesn't exist in Python 3.

Reply:
[__six.string_types__](http://six.readthedocs.io/#six.string_types) ;-)